whois-api/src/main/java/net/ripe/db/whois/api/rest/GeolocationService.java

```diff
@@ -116,7 +117,7 @@ public WhoisResources geolocation(
                     }
                 }
             } catch (EmptyResultDataAccessException ignored) {
-                // stop looking
+                LOGGER.debug("EmptyResultDataAccessException {} in {} class ", ignored.getMessage(), ignored.getClass().getName());
```

Comment:
This exception is expected, and is thrown by the lookup() method, if the RPSL object no longer exists. Do we need to log it at all ?